webui/src/components/ToolList.jsx

```diff
@@ -254,12 +254,16 @@ class ToolCard extends React.Component {
                     <div className="badges">
                         { !tool.authentication || tool.authentication == "no" ? null
                             : <div className="badge-holder auth" title="This tool requires a user account. Please check the Authentication information for more details.">
-                                <span className={"fa fa-key"} style={{padding:"6px 6px"}} aria-hidden="true"/>
+                                <span className={"fa fa-key"} aria-hidden="true"/>
                             </div> }
                         { !tool.standaloneApplication ? null
                             : <div className="badge-holder standalone" title="This tool requires local installation on one of your devices, please check the details.">
-                                <span className={"fa fa-download"} style={{padding:"6px 6px"}} aria-hidden="true"/>
+                                <span className={"fa fa-download"} aria-hidden="true"/>
                             </div> }
+                        { tool.webApplication && tool.webApplication.url && tool.webApplication.url.startsWith("http://") ?
+                            <div className="badge-holder unsafe" title="The connection to this tool is not secure. Unauthorized parties can spy on your data.">
```

Comment:
I think this phrasing can be understood as "your data" as in "your personal data", which is not what we mean here. I propose something more like: "...secure. Unknown 3rd parties can potentially access the resource while we transfer it to the tool"